sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClientTest.java

```diff
@@ -122,6 +130,43 @@ void teardown() {
         consumer.close();
     }
 
+    /**
+     * Verifies that this peek one messages.
+     */
+    @Test
+    void peekOneMessage() {
+        // Arrange
+        final int numberOfEvents = 1;
+        when(managementNode.peek())
+            .thenReturn(Mono.just(mock(ServiceBusReceivedMessage.class)));
+
+        // Act & Assert
+        StepVerifier.create(consumer.peek())
+            .expectNextCount(numberOfEvents)
+            .verifyComplete();
+
+    }
+
+    /**
+     * Verifies that this peek one messages from a sequence Number.
+     */
+    @Test
+    void peekWithSequenceOneMessage() {
+        // Arrange
+        final int numberOfEvents = 1;
+        final int fromSequenceNumber = 10;
+
+        when(managementNode.peek(fromSequenceNumber))
+            .thenReturn(Mono.just(mock(ServiceBusReceivedMessage.class)));
+
+        // Act & Assert
+        StepVerifier.create(consumer.peek(fromSequenceNumber))
+            .expectNextCount(numberOfEvents)
+            .verifyComplete();
+
+        Mockito.framework().clearInlineMocks();
```

Comment:
Should put this in AfterEach, if the test fails at line 163, it'll never be run.

Reply:
It is already there in `@AfterEach`